.changeset/dry-penguins-yell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-graph': patch
+'@finos/legend-manual-tests': patch
+'@finos/legend-studio': patch
+---
```

Comment:
we should have 2 changesets with this. 
one empty one with 
'@finos/legend-graph': patch
'@finos/legend-manual-tests': patch

and another with jus legend-studio and a brief description on what we addded

Reply:
More details on changeset splitting here - https://github.com/finos/legend-studio/blob/master/CONTRIBUTING.md#changeset